hazelcast/src/main/java/com/hazelcast/collection/impl/queue/operations/QueueOperation.java

```diff
@@ -99,6 +98,7 @@ public void afterRun() throws Exception {
 
     @Override
     public void beforeRun() throws Exception {
+        container = getOrCreateContainer();
```

Comment:
you are right, I'll have a look. 

by the way I'm about to push an update with slightly different logic. here is the idea:
1. in `beforeRun()` always initialises the `container` field
2. all other methods use `getContainer()` which just returns the content of the field.